### ISSUE TITLE
Waterfox addon - not working,  host upgrade available

### ISSUE THREAD

**Issue by gris2** [pdf/kdeconnect-chrome-extension#12]
Just installed the Chrome extension and the Firefox addon following the instructions. The Chrome extension works fine. The Firefox addon gives the following message:

> A host upgrade v0.1.3 is available, please follow the upgrade instructions
> No devices found...

I have downloaded the latest version of the host.

I am using Waterfox, not Firefox. Could that be the problem? I installed the file in Waterfox's native-messaging-hosts.

**Comment by pdf** [comment]
I certainly haven't tested Waterfox, but the fact that you're seeing that message suggests that the extension is at least talking to the native host.  This may sound obvious, but can you make sure you are using [v0.1.3](https://github.com/pdf/kdeconnect-chrome-extension/releases/tag/v0.1.3) of the host?

**Comment by gris2** [comment]
I am using the latest version: 

> kdeconnect-chrome-extension-v0.1.3-linux_amd64

Well I installed Firefox just to test it. The addon on Firefox didn't say anything about upgrading but it says "no devices available".

Strangely enough, once I installed the addon on Firefox, the one in Waterfox started working fine. It is very strange because Waterfox and Firefox have different folders. 

When Waterfox launches the addon shows the red exclamation mark for a bit but when I visit a page it just works fine.

FYI I am using Waterfox 55.2.2 and Firefox 52.4.0. My system is Opensuse Leap 42.3 with Plasma5. 

No idea what is going on but the important thing is that it works on Waterfox now. 

I think this is now solved. Thanks for your help.
